intermittent_tracker/templates/most_flaky.html

```diff
@@ -29,6 +30,13 @@
                 <td>{{ loop.index }}</td>
                 <td>{{ test.path }}</td>
                 <td>{{ test.total_unexpected_count }}</td>
+                <td>
+                    {% if test.issues %}
+                        <a href="https://github.com/servo/servo/issues/{{ test.issues.issues[0].number }}" rel=”noopener" target="_blank">{{ test.issues.issues[0].title }}</a>
+                    {% else %}
+                        No issues found
+                    {% endif %}
+                </td>
```

Comment:
I think this could easily make the table quite wide. Would it be possible to put a series of little links after the test name in the second column? Maybe just make the link title say something like "(issue)" or put an little GitHub icon. Whatever you like is fine by me, as long as the column isn't too wide.